### PR TITLE
400ng yearly import docs

### PR DESCRIPTION
**Pull request by garrettqmartin8**
## Description

Docs for importing 400ng data so we don't have to scramble as much next time!

### CODE REVIEW

File: docs/data/tariff400ng-yearly-import.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    puts '-- Seeding 2019 400NG baseline rates...'
+    Seeds::BaselineRates.new(
+      date_range: Range.new(Date.parse('2019-05-15'), Date.parse('2020-05-14')),
+      file_path: Rails.root.join('lib', 'data', '2019 400NG Baseline Rates.xlsx')
+    ).seed!
+    ```
+6. Run `rails db:reset` to drop the database, re-run migrations, and re-run the seeds import.
+7. Dump the tables: `pg_dump --inserts -t full_packs -t full_unpacks -t linehauls -t service_areas -t shorthauls move_mil_development`
+
+### Load dumped tables into database via migrations`
```
Comment (akostibas) [inline docs/data/tariff400ng-yearly-import.md]: Errant backtick at the end of the sentence

File: docs/data/tariff400ng-yearly-import.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## Obtain yearly rates `xlsx` file from ustranscom
+
+1. visit: [https://www.ustranscom.mil/dp3/hhg.cfm](https://www.ustranscom.mil/dp3/hhg.cfm) (for some reason, I had to load hit this url twice... the first visit redirected to another page).
+2. Look under “Special Requirements and Rates Team” -> “Domestic” -> “400NG Baseline Rates” and download yearly rate file.
+3. Copy the file to DP3 Google drive: DP3 -> Engineering -> Reference Docs -> rate engine: filing
+
+## Importing `full_pack_rates`, `full_unpack_rates`, `linehaul_rates`, `service_areas`, and `shorthaul_rates`
+
+### extract data from `xlsx` file via `Ruby` scripts
```
Comment (akostibas) [inline docs/data/tariff400ng-yearly-import.md]: Capitalize first `E`

File: docs/data/tariff400ng-yearly-import.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+#### related prs
+
+* [2018 data load](https://github.com/transcom/mymove/pull/338)
+* [2019 data load](https://github.com/transcom/mymove/pull/2036)
+
+1. create migration to load dump tables into `temp tables`. (see pr for example)
+2. create migration to transform `temp table` data into `MilMove` table structures. (see pr for example)
+
+## add additional `sit` data to `tariff400ng_service_areas` table
```
Comment (akostibas) [inline docs/data/tariff400ng-yearly-import.md]: Capitalize **A**dd

File: docs/data/tariff400ng-yearly-import.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+### The extracted data from the ruby scripts doesn't contain all the data we need. We also need `185A SIT First Day & Whouse`, `185B SIT Addl Days`, and `SIT PD Schedule` found on the `Geographical Schedule` sheet
+
+#### related prs
```
Comment (akostibas) [inline docs/data/tariff400ng-yearly-import.md]: Caps

File: docs/data/tariff400ng-yearly-import.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+4. ```select effective_date_lower, effective_date_upper, count(*) from tariff400ng_linehaul_rates group by effective_date_lower, effective_date_upper order by effective_date_lower DESC;```
+
+5. ```select effective_date_lower, effective_date_upper, count(*) from tariff400ng_service_areas group by effective_date_lower, effective_date_upper order by effective_date_lower DESC;```
+
+6. ```select effective_date_lower, effective_date_upper, count(*) from tariff400ng_shorthaul_rates group by effective_date_lower, effective_date_upper order by effective_date_lower DESC;```
+
+## Test
+
+1. Deploy branch in `experimental`.
+2. Create a move for a date that uses data loaded by these migrations and ensure the app doesn't crash.
```
Comment (akostibas) [inline docs/data/tariff400ng-yearly-import.md]: I'd specify that they should watch the logs for errors of any kind, not just crashes.

Bonus points if you can give more detail about what it means to "make a move that uses the data."

**Comment by lynzt** [comment]
@garrettqmartin8 I have another section to add based on findings yesterday so hold off on merging till then...

File: docs/data/tariff400ng-yearly-import.md (hunk 5 above)
Comment (lynzt) [reply docs/data/tariff400ng-yearly-import.md]: @akostibas updated and tried to be more clear. 
Also added another section for an addition transformation migration done for certain codes in `tariff400ng_item_rates`

**Comment by lynzt** [comment]
@garrettqmartin8 good to go now...

File: docs/data/tariff400ng-yearly-import.md (hunk 5 above)
Comment (akostibas) [reply docs/data/tariff400ng-yearly-import.md]: Thanks! Does this address that one record that didn't make it through in the first attempt at importing?

Comment (lynzt) [reply docs/data/tariff400ng-yearly-import.md]: No, the 1 missing record was added in this pr: https://github.com/transcom/mymove/pull/2051

This is due to data assumptions in the `item_rate` transform which we have to modify to fit our database and use case. Basically, we had to move the lower weight limit to 0 and `rate_cents` were in dollars vs cents for certain item codes...

**Pull request merged**